Kunena/Kunena-Forum
# cleanup folders in install script

Pull Request for Issue # . 
 
#### Summary of Changes 
added preflight folder cleanup. it is safe to remove folders in preflight as these will be then filled with the update or installation of the extension.
ASSUMPTION (!) here is that the site owner did not do any modifications / added files to the folders that are being deleted

It checks for the installed version and when that is above 6.0.0 and below 6.3.0 then the folders are handled.

#### Testing Instructions

build package with this PR in it and install it, there should not be any errors and everything should be working as it should.
All Kunena provided folders should now be exactly like in the install package without any left over files from previous versions that where not removed.
